src/openrct2/game.h

```diff
@@ -96,6 +96,12 @@ enum GAME_COMMAND {
 	GAME_COMMAND_COUNT
 };
 
+enum MISC_COMMAND {
+	MISC_COMMAND_CHAT = -1,
+	MISC_COMMAND_TOGGLE_SCENERY_CLUSTER = -2,
+	MISC_COMMAND_PASSWORDLESS_LOGIN = -3,
+};
```

Comment:
Is there any point in having these available in network-less builds?

These should rather be declared in guarded part in `network.h`.